config/dev.exs

```diff
@@ -24,7 +24,11 @@ config :code_fund, CodeFundWeb.Endpoint,
   ],
   ga_tracking_id: System.get_env("GA_TRACKING_ID"),
   metabase_site_url: System.get_env("METABASE_SITE_URL"),
-  metabase_secret_key: System.get_env("METABASE_SECRET_KEY")
+  metabase_secret_key: System.get_env("METABASE_SECRET_KEY"),
+  slack_token: System.get_env("SLACK_TOKEN"),
+  slack_bot_name: System.get_env("SLACK_BOT_NAME"),
```

Comment:
Can we pull these all into a dependency specific configuration rather than environment specific ones? Check `config/configs`

Reply:
no problem, but then what is the better way to disable slack in test environment?

Reply:
I think we can do a simple module that implements GenServer and the start_link function there calls start_link on Slack.Bot if the token is set in System environment.